charts/corda-dev/kafka-config/server.properties

```diff
@@ -0,0 +1,24 @@
+listeners=SASL_SSL://:9092,CONTROLLER://:9093,SASL_PLAINTEXT://:9094
+advertised.listeners=SASL_SSL://{{ include "corda-dev.kafkaName" . }}:9092,SASL_PLAINTEXT://localhost:9094
+inter.broker.listener.name=SASL_SSL
```

Comment:
What happens if sasl is not enabled? but your left with tls?

Reply:
Unlike the original chart, there aren't options to disable TLS or auth - you always get both. That's the sort of thing I was hoping to solicit feedback on but, having had none, I'm going to keep it as simple as possible!

Reply:
ssl.key password, can we avoid hardcoding this?

Reply:
To be honest, I would have generated the keystore without a passphrase at all (given that both the keystore and password only exist within the container so if you have access to one, you have access to the other) but Kafka didn't seem to like not having one.

Reply:
Pushed an update to generate the keystore password.